content/en/llm_observability/setup/sdk/nodejs.md

```diff
@@ -0,0 +1,711 @@
+---
+title: LLM Observability Node.js SDK Reference
+code_lang: nodejs
+type: multi-code-lang
+code_lang_weight: 30
+aliases:
+    - /tracing/llm_observability/sdk/nodejs
+    - /llm_observability/sdk/nodejs
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">LLM Observability is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Overview
+
+The LLM Observability SDK for Node.js enhances the observability of your Javascript-based LLM applications. The SDK supports Node.js versions 16 and newer. For information about LLM Observability's integration support, see [Auto Instrumentation][5].
+
+You can install and configure tracing of various operations such as workflows, tasks, and API calls with wrapped functions or traced blocks. You can also annotate these traces with metadata for deeper insights into the performance and behavior of your applications, supporting multiple LLM services or models from the same environment.
+
+## Setup
+
+### Prerequisites
+
+1. The latest `dd-trace` package must be installed:
+
+{{< code-block lang="shell">}}
+npm install dd-trace
+{{< /code-block >}}
+
+2. LLM Observability requires a Datadog API key (see [the instructions for creating an API key][1]).
+
+### Command-line setup
+
+Enable LLM Observability by running your application with `NODE_OPTIONS="--import dd-trace/initialize.mjs"`  and specifying the required environment variables.
+
+**Note**: `dd-trace/initialize.mjs` automatically turns on all APM integrations.
+
+{{< code-block lang="shell">}}
+DD_SITE=<YOUR_DATADOG_SITE> DD_API_KEY=<YOUR_API_KEY> DD_LLMOBS_ENABLED=1 \
+DD_LLMOBS_ML_APP=<YOUR_ML_APP_NAME> NODE_OPTIONS="--import dd-trace/initialize.mjs" node <YOUR_APP_ENTRYPOINT>
+{{< /code-block >}}
+
+`DD_API_KEY`
+: required - _string_
+<br />Your Datadog API key.
+
+`DD_SITE`
+: required - _string_ 
+<br />The Datadog site to submit your LLM data. Your site is {{< region-param key="dd_site" code="true" >}}.
```

Comment:
this is awesome that we have the site here. Can we also put it in the codeblock to auto-fill it in? One less step for the user.

Reply:
yes, good idea!